source/reference/lists.txt

```diff
@@ -0,0 +1,140 @@
+..  _style-guide-lists:
+
+=====
+Lists
+=====
+
+.. default-domain:: mongodb
+
+You can create *ordered* or *unordered* lists in {+rst+}.
+
+Ordered Lists
+-------------
+
+Ordered lists show a number for each item in the list. Create an ordered list by
+starting a line with ``#.``, or by using a number followed by a ``.``. Using
+``#.`` in place of numbers prompts Snooty to generate list numbers automatically.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Markup
+     - Result
+
+   * -
+
+       .. code-block:: rst
+          :copyable: false
+
+           #. Item 1
+           #. Item 2
+           #. Item 3
+
+     - #. Item 1
+       #. Item 2
+       #. Item 3
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Markup
+     - Result
+
+   * -
+
+       .. code-block:: rst
+          :copyable: false
+
+           1. Item 1
+           2. Item 2
+           3. Item 3
+
+     - 1. Item 1
+       2. Item 2
+       3. Item 3
+
+Unordered Lists
+---------------
+
+Unordered lists show a bullet point for each item in the list. Create an
+unordered list by starting a line with ``*`` or ``-``. 
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Markup
+     - Result
+
+   * -
+
+       .. code-block:: rst
+          :copyable: false
+
+           * Item 1
+           * Item 2
+           * Item 3
+
+     - * Item 1
+       * Item 2
+       * Item 3
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Markup
+     - Result
+
+   * -
+
+       .. code-block:: rst
+          :copyable: false
+
+           - Item 1
+           - Item 2
+           - Item 3
+
+     - - Item 1
+       - Item 2
+       - Item 3
+
+Nested Lists
+------------
+
+You can create nested lists with both ordered and unordered lists. Create a
+nested list by first creating a list, then creating a second list separated by
+a blank line and indented. Indent the second list to the same level of
+the text of the parent list.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Markup
+     - Result
+
+   * -
+
+       .. code-block:: rst
+          :copyable: false
+
+           - Item 1
+
+             - Nested Item 1
+             - Nested Item 2
+             
+           - Item 2
+           - Item 3
+
+     - - Item 1
+
+         - Nested Item 1
+         - Nested Item 2
+
+       - Item 2
+       - Item 3
```

Comment:
Suggestion: For all of the code blocks in this PR, the code block is duplicated -- once as markup, once as the result. In my procedures PR, I use a single RST include file (once as a `literalinclude` for the literal code, once as an `include` for the rendered result) for both. That seems easier to keep up-to-date in the future -- would you consider switching to that implementation here?